cpp/src/arrow/util/cpu_info.h

```diff
@@ -101,6 +107,9 @@ class ARROW_EXPORT CpuInfo {
   /// Returns the model name of the cpu (e.g. Intel i7-2600)
   std::string model_name();
 
+  /// Returns whether or not the cpu is provided by this vendor.
+  bool IsVendor(Vendor vendor) const { return vendor == vendor_; }
```

Comment:
It would seem more useful to expose the vendor directly:
```c++
Vendor vendor() const { ... }
```

Reply:
done